### PR TITLE
Perform some minor code cleanup

### PR DESCRIPTION
**Pull request by Holzhaus**
These a some of the issues stumbled over while testing PR #2464.

### CODE REVIEW

File: res/skins/Tango/deck_overview.xml [hunk 1]
```diff
@@ -43,13 +43,13 @@ Variables:
     </MarkRange>
     <Mark>
       <Control>loop_start_position</Control>
-      <Text>[&nbsp;</Text>
+      <Text>[&#x0A;</Text>
```
Comment (daschuer) [inline res/skins/Tango/deck_overview.xml]: Please add a comment here, that we want &nbsp and why we can't use it and probably why we need it here.

I think we need to choose 
&#xa0 or &#160

File: res/skins/Tango/deck_overview.xml [hunk 2]
```diff
@@ -43,13 +43,13 @@ Variables:
...
       <Align>top|left</Align>
       <TextColor>#FFFFFF</TextColor>
     </Mark>
     <Mark>
       <Control>loop_end_position</Control>
-      <Text>&nbsp;]</Text>
+      <Text>&#x0A;]</Text>
```
Comment (daschuer) [inline res/skins/Tango/deck_overview.xml]: The same here

**Review by daschuer — CHANGES_REQUESTED** [review]

File: res/skins/Tango/deck_overview.xml (hunk 1 above)
Comment (Holzhaus) [reply res/skins/Tango/deck_overview.xml]: Whoops, sorry, I must have switched the numbers by mistake. I fixed up the commit.

**Comment by Holzhaus** [comment]
Merge?

**Review by daschuer — APPROVED** [review]

**Comment by daschuer** [comment]
LGTM, thank you.

**Pull request merged**